docs/adr/0002-use-rspack-module-federation.md

```
@@ -76,13 +81,15 @@ All remotes must share:
 - `react-router` (singleton)
 - `@patternfly/react-core` (singleton)
 
+`OdhFederationPlugin` applies this policy from the `package.json` in rspack `compiler.context`. Do not maintain a manual shared map for those packages.
+
 ## Alternatives Considered
 
 ### Single-Spa
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)